### PR TITLE
Adding the Routine to plot the Tropical Atlantic Region to the main repository

### PR DESCRIPTION
**Pull request by anaclarabmariani**

### CODE REVIEW

File: Exploring_Data_NCAR.ipynb [hunk 1]
```diff
@@ -0,0 +1,2073 @@
+{
```
Comment (paigem) [inline Exploring_Data_NCAR.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    ds = cat['CESM_POP_hires_RCP8_5'].to_dask()
</pre><p>Note that this line indicates that you are loading in the RCP8.5 scenario - which is the climate change run of this NCAR model. The "control" is the model run without climate change forcing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by NoraLoose** [comment]
Amazing work @anaclarabmariani! I only had a few small suggestions for your MKE-EKE notebook. You can see them best by clicking on the purple ReviewNB button in [this comment](https://github.com/oceanhackweek/ohw22-proj-gcmfilters/pull/10#issuecomment-1220046319).

**Comment by anaclarabmariani** [comment]
<p>Great idea! I think I'll add a markdown about that just before this cell.</p>

---

 <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/10/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by anaclarabmariani** [comment]
Upload the new version! Thank you so much @NoraLoose and @paigem !

**Comment by paigem** [comment]
@anaclarabmariani It looks like you got the `cartopy` plots working!! Well done! 👏

**Pull request merged**